tests/test_timing/import_tidy3d.py

```diff
@@ -1,17 +1,17 @@
-import subprocess
-import time
-
-n = 100
-python_load_time = 0
-tidy3d_load_time = 0
-
-for _ in range(n):
-    s = time.time()
-    subprocess.call(["python", "-c", "import tidy3d"])
-    tidy3d_load_time += time.time() - s
-
-    s = time.time()
-    subprocess.call(["python", "-c", "pass"])
-    python_load_time += time.time() - s
-
-print(f"average tidy3d load time = {(tidy3d_load_time - python_load_time) / n}")
+# import subprocess
+# import time
+#
+# n = 100
+# python_load_time = 0
+# tidy3d_load_time = 0
+#
+# for _ in range(n):
+#     s = time.time()
+#     subprocess.call(["python", "-c", "import tidy3d"])
+#     tidy3d_load_time += time.time() - s
+#
+#     s = time.time()
+#     subprocess.call(["python", "-c", "pass"])
+#     python_load_time += time.time() - s
+#
+# print(f"average tidy3d load time = {(tidy3d_load_time - python_load_time) / n}")
```

Comment:
Why?

Reply:
Whoops that was temporary to speed up testing, but now have brought it back.